deploy/util/io.py

```diff
@@ -0,0 +1,23 @@
+
+
+import logging
+
+def firesim_input(prompt: object = None) -> str:
+    """wrap __builtin__.input() understanding the idiocyncracies of firesim+fabric+logging
+
+    Log the prompt at CRITICAL level so that it will go to the terminal and the log.
+    Log the entered text as DEBUG so that the log contains it.
+    Don't pass the prompt to __builtin__.input() because we don't need StreamLogger to also
+    be trying to log the prompt.
+    
+    See 'streamlogger.py' and it's use at the end of 'firesim.py'
```

Comment:
```suggestion
    """wrap builtins.input() understanding the idiocyncracies of firesim+fabric+logging

    Log the prompt at CRITICAL level so that it will go to the terminal and the log.
    Log the entered text as DEBUG so that the log contains it.
    Don't pass the prompt to builtins.input() because we don't need StreamLogger to also
    be trying to log the prompt.
    
    See 'streamlogger.py' and it's use at the end of 'firesim.py'
```

Update docstring for firesim_input to correctly point to `builtins` not `__builtin__`...